crates/ink/Cargo.toml

```diff
@@ -1,38 +1,40 @@
 [package]
 name = "ink"
-version = "4.2.0"
-authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
-edition = "2021"
-rust-version = "1.63"
 
-license = "Apache-2.0"
-readme = "README.md"
-repository = "https://github.com/paritytech/ink"
-documentation = "https://docs.rs/ink"
-homepage = "https://www.parity.io/"
+edition.workspace = true
+categories.workspace = true
+homepage.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
+authors = ["Parity Technologies <admin@parity.io>", "Robin Freyler <robin@parity.io>"]
```

Comment:
It seems `authors` are the same, maybe we can use `workspace` too?

Reply:
no, they are not the same across crates and I really don't want to get into authorship resolution here - I don't know what is the legal status of it

Reply:
Only 10 of 57 are unique. For them, you can use a custom variant
<img width="634" alt="image" src="https://github.com/paritytech/ink/assets/18346821/48d2163d-b7bd-4064-b7f8-32eeede2aa8d">

Reply:
there are 14 crates in workspace, only 3 of them have this basic authorship; the rest that you have grepped are in integration-tests; nevertheless, suggestion applied

Reply:
Hmm, maybe you forgot to push or maybe I'm reviewing too earlier=D

Reply:
yes, that's the case, sorry

Reply:
pushed